src/System.Net.Primitives/tests/FunctionalTests/Cookie.cs

```diff
@@ -0,0 +1,344 @@
+﻿using System;
+using System.Net;
+
+using Xunit;
+
+namespace System.Net.Primitives.Functional.Tests
+{
+    public static class CookieTest
+    {
+        [Fact]
+        public static void TestConstructors()
+        {
+            Cookie c = new Cookie();
+
+            c = new Cookie("hello", "goodbye");
+            Assert.Equal("hello", c.Name);
+            Assert.Equal("goodbye", c.Value);
+
+            c = new Cookie("hello", "goodbye", "foo");
+            Assert.Equal("hello", c.Name);
+            Assert.Equal("goodbye", c.Value);
+            Assert.Equal("foo", c.Path);
+
+            c = new Cookie("hello", "goodbye", "foo", "bar");
+            Assert.Equal("hello", c.Name);
+            Assert.Equal("goodbye", c.Value);
+            Assert.Equal("foo", c.Path);
+            Assert.Equal("bar", c.Domain);
+        }
+
+        [Fact]
+        public static void TestComment()
+        {
```

Comment:
+ 1 on the naming convention. We would like the tests to have an uniform naming convention.